build.sh

```diff
@@ -10,42 +10,6 @@ TOOLS_DIR=$SCRIPT_DIR/tools
 NUGET_EXE=$TOOLS_DIR/nuget.exe
 CAKE_EXE=$TOOLS_DIR/Cake/Cake.exe
 
-# Define default arguments.
-SCRIPT="build.cake"
-TARGET="Travis"
-CONFIGURATION="Release"
-VERBOSITY="verbose"
-DRYRUN=false
-SHOW_VERSION=false
-
-# Parse arguments.
-for i in "$@"; do
-    case $1 in
-        -s|--script) SCRIPT="$2"; shift ;;
-        -t|--target) TARGET="$2"; shift ;;
-        -c|--configuration) CONFIGURATION="$2"; shift ;;
-        -v|--verbosity) VERBOSITY="$2"; shift ;;
-        -d|--dryrun) DRYRUN=true ;;
-        --version) SHOW_VERSION=true ;;
-    esac
-    shift
-done
-
-# Make sure the tools folder exist.
-if [ ! -d $TOOLS_DIR ]; then
```

Comment:
The tools things should not have been removed, I was over-eager on the dd :) I've uploaded a new patch